proto/privval/types.proto

```diff
@@ -18,10 +18,19 @@ message FilePVKey {
 // FilePVLastSignState stores the mutable part of PrivValidator.
 message FilePVLastSignState {
   int64 height     = 1;
-  int64 round      = 2;
+  int32 round      = 2;
   int32 step       = 3;
   bytes signature  = 4;
   bytes sign_bytes = 5;
 
   string file_path = 6;
 }
+
+enum Errors {
+  ERRORS_UNKNOWN             = 0;
```

Comment:
drop the `ERRORS` prefix?

Reply:
following the protobuf style guide, this will have to be done else where as well (ABCI).